pkg/providers/oracle/model/objectstore/objectstore.go

```diff
@@ -1,25 +1,27 @@
 package objectstore
 
 import (
-	pipelineAuth "github.com/banzaicloud/pipeline/auth"
+	"github.com/banzaicloud/pipeline/auth"
 )
 
 // TableName constants
 const (
-	ManagedOracleBucketTableName = "managed_oracle_buckets"
+	BucketsTableName = "oracle_buckets"
 )
 
-// ManagedOracleBucket is the schema for the DB
-type ManagedOracleBucket struct {
-	ID            uint                      `gorm:"primary_key"`
-	Organization  pipelineAuth.Organization `gorm:"foreignkey:OrgID"`
-	OrgID         uint                      `gorm:"index;not null"`
-	CompartmentID string
-	Name          string `gorm:"unique_index:bucketNameLocation"`
-	Location      string `gorm:"unique_index:bucketNameLocation"`
+// ObjectStoreBucket is the schema for the DB
+type ObjectStoreBucket struct {
```

Comment:
Either follow the `ObjectStoreModel` or rename all models. I'm in favor of the first option.

Reply:
I would probably move this model to the `github.com/banzaicloud/pipeline/pkg/providers/oracle` package as I did with other models.

Reply:
Currently the model is in a separate package, the `ObjectStoreModel` name seemed redundant.

Also `ObjectStoreModel` isn't as specific as it should be. We store bucket info there, so it should be `ObjectStoreBucket` or `ObjectStoreBucketModel` if it is not in a separate package.